src/library/features/crates/cratetreemodel.cpp

```diff
@@ -0,0 +1,61 @@
+#include <QVector>
+
+#include "library/trackcollection.h"
+#include "library/features/crates/cratetreemodel.h"
+
+#include "util/db/dbid.h"
+
+namespace {
+
+const CrateId kRootId(-1);
```

Comment:
The -1 is unnecessary, just use a default constructed CrateId. General rule: Don't use literal id values anywhere.